web/src/pages/Route/locales/zh-CN.ts

```diff
@@ -125,4 +125,8 @@ export default {
   'page.route.TabPane.authentication': '认证',
   'page.route.TabPane.response': '响应结果',
   'page.route.debugWithoutAuth': '当前请求不启用任何认证方式。',
+  'page.route.button.exportOpenApi': '导出 OpenApi',
+  'page.route.exportRoutesTips': '请选择下载文件的类型',
+  'page.route.button.importOpenApi': '导入 OpenApi',
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/104534878-0ff4b580-5650-11eb-9aa1-b7878785aa64.png)

```suggestion
  'page.route.button.importOpenApi': '导入 OpenAPI',
```